plugins/metrics/src/main/java/org/apache/cloudstack/response/HostMetricsResponse.java

```diff
@@ -52,10 +52,6 @@ public class HostMetricsResponse extends HostResponse {
     @Param(description = "the total cpu allocated in Ghz")
     private String cpuAllocated;
 
-    @SerializedName("cpuloadaverage")
```

Comment:
Already there in HostResponse.

Reply:
yes, it has the same serializedName with `cpuloadaverage` in HostResponse

```
    @SerializedName(ApiConstants.CPU_LOAD_AVERAGE)
    @Param(description = "the cpu average load on the host")
    private Double cpuloadaverage;
```